src/routes/user/[name=validname]/index.svelte

```diff
@@ -15,11 +15,19 @@
         </div>
     </div>
     {:else if result == 'normal'}
-    <div class="w-screen h-screen grid place-items-center text-white bg-gray-600 p-4">
+    <div class="w-screen h-screen grid place-items-center text-white bg-amber-600 p-4">
```

Comment:
this is a good color ngl although maybe it should be used for botted instead of normal

Reply:
it doesn't really matter though